README.md

```diff
@@ -25,7 +25,7 @@ See the wiki for detailed documentation.
 Ensure that docker is up to date and initialized.
 Spin up mysql locally:
 ```
-docker run -p 3306:3306 --name mysql -e MYSQL_ROOT_PASSWORD=rawls-test -e MYSQL_USER=rawls-test -e MYSQL_PASSWORD=rawls-test -e MYSQL_DATABASE=testdb -d mysql/mysql-server:5.7
+docker run -p 3306:3306 --name mysql -e MYSQL_ROOT_PASSWORD=rawls-test -e MYSQL_USER=rawls-test -e MYSQL_PASSWORD=rawls-test -e MYSQL_DATABASE=testdb -d mysql/mysql-server:5.7 --max_connections=10000
```

Comment:
This same line is used in the jenkins build - will need to make sure it's there too.